src/env.json

```diff
@@ -3,7 +3,7 @@
   "gethWebsocketsURL": "ws://127.0.0.1:8546",
   "augurNodeUrl": "ws://127.0.0.1:9001",
   "networkID": null,
-  "autoLogin": false,
+  "autoLogin": true,
```

Comment:
Don’t commit changes made to env.json

Reply:
Woops, oversight, I'll make a checklist so i don't forget this next time